arch/arm64/boot/dts/broadcom/bcm2711-rpi-cm4-fr201.dts

```diff
@@ -0,0 +1,4 @@
+#include "../../../../arm/boot/dts/bcm2711-rpi-cm4.dts"
```

Comment:
It's better to put this include inside the [arch/arm/boot/dts/bcm2711-rpi-cm4-fr201.dts](https://github.com/lf-edge/eve-kernel/pull/156/commits/05ba1d855077a8f6e97ef5649f82f04d7edcb4d4#diff-ee9a61157fa80e5199b59c2ffcf04f74ad1affb2c57ffad11b45dcd19c43a5d6), since it's a dependency for it.... and here you can let only the second include....

Reply:
Done